test/lib/post.js

```diff
@@ -35,11 +35,25 @@ function request(config, data) {
       });
     });
     request.once('error', reject);
-    request.write(Object.keys(data).map(key => `${key}=${data[key]}`).join('&'));
+    if (data) {
+      request.write(Object.keys(data).map(key => `${key}=${data[key]}`).join('&'));
+    }
     request.end();
   });
 }
 
+/**
+ * Make an HTTP(S) get request.
+ * @param {*} config
+ * @returns {Promise<*>}
+ */
+function get(config) {
+  return request(Object.assign({
+    method: 'GET'
+  }, config));
```

Comment:
nit: I noticed we're still using `Object.assign(..` everywhere. Should we start using spread operator?
```js
  return request({
    method: 'GET',
    ...config,
  });
```

Reply:
I think we should start using spread (and lot more) once we start moving to typescript. For now I will keep Object.assign to be consistant with rest of the file.